README.md

```diff
@@ -118,6 +118,80 @@ Example for a subject like `Foo::Bar#baz` it will run all example groups with de
 `Foo::Bar#baz`, `Foo::Bar` and `Foo`. The order is important, so if mutant finds example groups in the
 current prefix level, these example groups *must* kill the mutation.
 
+Reading Mutation Output
```

Comment:
Or better "Reading Reports", because its clear we talk about Mutant in this readme. So there is no need to repeat this fact in a heading.